pkg/outputs/nats_outputs/nats/nats_output.go

```diff
@@ -145,7 +146,7 @@ func (n *NatsOutput) Init(ctx context.Context, name string, cfg map[string]inter
 		return err
 	}
 
-	n.msgChan = make(chan *outputs.ProtoMsg)
+	n.msgChan = make(chan *outputs.ProtoMsg, uint(n.Cfg.BufferSize))
```

Comment:
this type conversion is not necessary anymore.

Reply:
Good, catch, this has been updated.